backend/models.py

```diff
@@ -787,25 +754,25 @@ def get_usage(self, user: User, limit: str | QuotaLimit):
 
 class QuotaIncreaseRequest(models.Model):
     class StatusTypes(models.TextChoices):
-        PENDING = "pending"
-        APPROVED = "approved"
-        REJECTED = "rejected"
+        PENDING = "pending", "Pending"
+        APPROVED = "approved", "Approved"
+        REJECTED = "rejected", "Rejected"
 
     user = models.ForeignKey(User, on_delete=models.CASCADE)
     quota_limit = models.ForeignKey(QuotaLimit, on_delete=models.CASCADE, related_name="quota_increase_requests")
-    reason = models.CharField(max_length=1000)
+    reason = models.TextField(max_length=1000)
```

Comment:
In django it's good practice if using a Max Length to use CharField, a TextField isnt validated at all I dont think